.github/workflows/publish-indy.yml

```diff
@@ -0,0 +1,99 @@
+name: Publish ACA-Py (Indy)
+on:
+  release:
+    types: [released]
+  workflow_dispatch:
+    inputs:
+      indy_version:
+        description: 'Indy SDK Version'
+        required: false
+        type: string
+      tag:
+        description: 'Image tag'
+        required: true
+        type: string
+      version:
+        description: "Version label in image"
+        required: true
+        type: string
+
+env:
+  INDY_VERSION: 1.16.0
+
+jobs:
+  publish-image:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+
+    name: Publish (Indy)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Gather image info
+        id: info
+        run: |
+          echo "::set-output name=repo-owner::${GITHUB_REPOSITORY_OWNER,,}"
+
+      - name: Cache Docker layers
+        uses: actions/cache@v3
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Log in to the GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Using `github.actor` as the username causes issues in the Hyperledger org.  Use `github.repository_owner` instead.  For example; https://github.com/hyperledger/indy-shared-gha/blob/main/.github/workflows/buildimage.yaml#L51-L53